src/GafferSceneUI/LightPositionTool.cpp

```diff
@@ -0,0 +1,969 @@
+
+	// Point in the pivot-shadowTarget direction, in local space
+	const V3f targetZAxis = ( shadowTarget - shadowPivot ) * worldParentTransformInverse;
+
+	return rotationMatrixWithUpDir( V3f( 0.f, 0.f, -1.f ), targetZAxis, currentYAxis );
+
+}
+
+class ShadowHandle : public Handle
+{
+
+	public :
+
+		ShadowHandle()
+		{
+
+		}
+
+		~ShadowHandle() override
+		{
+
+		}
+
+		// Set the position of the shadow pivot from the given world-space coordinate.
+		void setShadowPivot( const std::optional<V3f> &p )
+		{
+			m_shadowPivot = p;
+			dirty( DirtyType::Render );
+		}
+
+		std::optional<V3f> getShadowPivot() const
+		{
+			return m_shadowPivot;
+		}
+
+		// Set the position of the shadow target from the given world-space coordinate.
+		void setShadowTarget( const std::optional<V3f> &p )
+		{
+			m_shadowTarget = p;
+			dirty( DirtyType::Render );
+		}
+
+		std::optional<V3f> getShadowTarget() const
+		{
+			return m_shadowTarget;
+		}
+
+		void setPivotDistance( const std::optional<float> d )
+		{
+			m_pivotDistance = d;
+		}
+
+		std::optional<float> getPivotDistance() const
+		{
+			return m_pivotDistance;
+		}
+
+	protected :
+
+		void renderHandle( const Style *style, Style::State state ) const override
+		{
+			std::optional<V3f> shadowPivot = getShadowPivot();
+			std::optional<V3f> shadowTarget = getShadowTarget();
+			if( !shadowPivot && !shadowTarget )
+			{
+				return;
+			}
+
+			float lineRadius = 0;
+			float circleSize = 0;
+			float coneSize = 0;
+
+			const bool highlighted = state == Style::State::HighlightedState;
+			const bool selectionPass = (bool)IECoreGL::Selector::currentSelector();
+
+			if( selectionPass )
+			{
+				lineRadius = g_lineSelectionWidth;
+				circleSize = g_circleHandleSelectionWidth;
+				coneSize = g_arrowHandleSelectionSize;
+			}
+			else
+			{
+				lineRadius = highlighted ? g_lineHandleWidthLarge : g_lineHandleWidth;
+				circleSize = highlighted ? g_circleHandleWidthLarge : g_circleHandleWidth;
+				coneSize = highlighted ? g_arrowHandleSizeLarge : g_arrowHandleSize;
+			}
+
+			State::bindBaseState();
+			auto glState = const_cast<State *>( State::defaultState() );
+
+			IECoreGL::GroupPtr group = new IECoreGL::Group;
+
+			group->getState()->add(
+				new IECoreGL::ShaderStateComponent(
+					ShaderLoader::defaultShaderLoader(),
+					TextureLoader::defaultTextureLoader(),
+					"",
+					"",
+					constantFragSource(),
+					new CompoundObject
+				)
+			);
+
+			auto standardStyle = runTimeCast<const StandardStyle>( style );
+			assert( standardStyle );
+			const Color3f highlightColor3 = standardStyle->getColor( StandardStyle::Color::HighlightColor );
+			const Color4f highlightColor4 = Color4f( highlightColor3.x, highlightColor3.y, highlightColor3.z, 1.f );
+
+			group->getState()->add(
+				new IECoreGL::Color(
+					enabled() ? ( highlighted ? g_lightToolHighlightColor4 : highlightColor4 ) : g_lightToolDisabledColor4
+				)
+			);
```

Comment:
I reversed the colors in https://github.com/GafferHQ/gaffer/pull/5558/commits/518ce36a690ce11044c8ac846dea781c987ca46a. I think it is better in this, reversed, way. The handle matches the light and makes that visual connection, and the blue hover state makes the drag-ability more obvious.

My recollection of the LightTool coloring is that making the handles yellow made them hard to see the change when the tool is activated, making it harder to realize you could drag the handles. Especially for the spot light angle tools, where the dot icons are fairly subtle.

The quad light handles are maybe easier to understand since they have arrows / cones at the edges.

But I think that decision also came at a time when we didn't have the tooltip that follows the mouse for the light tool. So maybe after this is sorted it's worth revisiting to see how a color reversal there looks.